doc/Language/classtut.pod6

```diff
@@ -853,6 +910,43 @@ say $o.^parents.map({ $_.^name }).join(', ');
 Finally C<$o.^name> calls the C<name> method on the metaobject, which
 unsurprisingly returns the class name.
 
+Given an object C<$mp3> and the C<MP3TagData>class definition from the section
+L<Private methods|/language/classtut#Private_methods>, we can inquire about its
+public methods with C<.^methods>:
+
+=begin code :skip-test<compile time error>
+my $mp3 = MP3TagData.new(filename => 'football-head.mp3');
+say $mp3.^methods(:local);
+# OUTPUT: (TWEAK filename title artist album year comment genre track version
+# type Submethod+{is-hidden-from-backtrace}.new)
+=end code
+
+X<|^methods>
+C<$mp3.^methods(:local)> produces a list of L<Method|/type/Method>s that can be
+called on C<$mp3>. The C<:local> named argument limits the returned methods to
+those defined in the C<MP3TagData> class and excludes the inherited methods;
+C<MP3TagData> inherits from no class, so providing C<:local> makes no difference.
+
+To check if a type object (or an instance object) implements a certain public
```

Comment:
Then ^find_method is not indexed. While you're at it, you could also add an index entry (and explanation, as above)

Reply:
Do you mean here specifically, or within the whole documentation?

Reply:
Possibly here, unless you find a better place.